core/src/main/scala/org/apache/spark/deploy/master/Master.scala

```diff
@@ -765,7 +765,8 @@ private[spark] class Master(
       val ui = SparkUI.createHistoryUI(new SparkConf, replayBus, new SecurityManager(conf),
         appName + status, HistoryServer.UI_PATH_PREFIX + s"/${app.id}")
       try {
-        replayBus.replay(logInput, eventLogFile)
+        replayBus.replay(logInput, eventLogFile, 
+          eventLogFile.endsWith(EventLoggingListener.IN_PROGRESS))
```

Comment:
Final nit, and this is up to you; I think this could be clearer if the new value were written out as a `val maybeTruncated` first to describe what this is supposed to communicate.